ci/builders/linux_host_engine.json

```diff
@@ -0,0 +1,174 @@
+{
+    "builds": [
+        {
+            "archives": [],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "gclient_custom_vars": {
+                "download_android_deps": false
+            },
+            "gn": [
+                "--runtime-mode",
+                "debug",
+                "--unoptimized",
+                "--prebuilt-dart-sdk"
+            ],
+            "name": "host_debug_unopt",
+            "ninja": {
+                "config": "host_debug_unopt",
+                "targets": [
+                    "flutter",
+                    "flutter/sky/packages",
+                    "flutter/lib/spirv/test/exception_shaders:spirv_compile_exception_shaders"
+                ]
+            },
+            "tests": []
+        },
+        {
+            "archives": [
+                {
+                    "base_path": "out/host_debug/zip_archives/",
+                    "include_paths": [
```

Comment:
@christopherfujino this is how archiving will look like once we migrate to engine_v2 recipes.

Reply:
cool